rclcpp/include/rclcpp/executor.hpp

```diff
@@ -470,6 +470,7 @@ class Executor
   void
   execute_any_executable(AnyExecutable & any_exec);
 
+public:
```

Comment:
Why is this necessary?

Reply:
Same as below I don't derive from rclcpp::Executor

Reply:
removed